src/components/vegetation-graph.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useEffect, useState } from "react";
+import { observer } from "mobx-react";
+import { useStores } from "../use-stores";
+import { Chart as ChartJS, CategoryScale, LinearScale, PointElement, LineElement, BarElement, Title, Tooltip, Filler, Legend, defaults } from "chart.js";
+import { Line, Bar } from "react-chartjs-2";
+import { Vegetation } from "../types";
+
+import cssExports from "./common.scss";
+
+defaults.font.family = cssExports.robotoFontFamily;
+defaults.color = cssExports.controlGray;
+defaults.font.size = 14;
+
+ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, BarElement, Title, Tooltip, Filler, Legend);
+
+export const defaultOptions = {
+  responsive: true,
+  animation: {
+    duration: 0,
+  },
+  plugins: {
+    legend: {
+      display: false
+    },
+    title: {
+      display: true,
+      text: "Vegetation vs. Time",
+      fullSize: false, // center over chart area only
+      font: {
+        size: 15,
+        weight: "bold"
+      }
+    },
+  },
+  categoryPercentage: 1, // no spacing between bars
+  barPercentage: 0.9,
+  scales: {
+    x: {
+      ticks: {
+        maxTicksLimit: 6,
+      },
+      grid: {
+        display: false
+      },
+      border: {
+        color: cssExports.controlGray,
+      },
+      stacked: true
+    },
+    y: {
+      stacked: true,
+      min: 0,
+      max: 100,
+      ticks: {
+        stepSize: 20,
+      },
+      grid: {
+        display: false
+      },
+      border: {
+        color: cssExports.controlGray,
+      },
+      title: {
+        display: true,
+        text: "Vegetation (%)"
+      }
+    }
+  }
+};
+
+const datasetOptions = {
+  fill: true,
+  pointRadius: 0,
+  borderWidth: 0,
+};
+
+const RECENT_YEARS = 26;
+const GRAPH_HEIGHT = 210;
+
+export interface IProps {
+  allData?: boolean;
+}
+
+export const VegetationGraph: React.FC<IProps> = observer(({ allData }) => {
+  const { simulation } = useStores();
+  const [, setFontsReady] = useState<number>(0); // used to force re-render graph after fonts are ready.
```

Comment:
Cool - I also didn't know you could leave an array element empty when destructuring.  This is just to force a re-render right since you are not using the value?

Reply:
Yes, exactly, and if it was `[fontsReady, setFontsReady]`, ESLint would complain about `fontsReady` not being used.

The issue is that Chart.js draws everything on canvas and it happens before "Roboto Condensed" font is downloaded. As a result, the initial render uses the fallback font. So, when the user is looking at the empty graph, the fonts would be off.